app/models/feature_flag.rb

```diff
@@ -4,16 +4,20 @@ class FeatureFlag < ApplicationRecord
 
   # Class methods
 
+  # Returns true if all valid, else false
   def self.update_all_flags( params )
+    all_valid = true
     params['flags'].keys.map do |flag_id|
       flag = find( flag_id )
       flag_params = params['flags'][flag_id]
-      _f = flag.update(
```

Comment:
Rubocop was trying to tell us something!

Reply:
Hah! Convergent evolution!  https://github.com/denny/ShinyCMS-ruby/commit/f5e490347d522c65ae3c4a8c7816fc3ec84448f4